demo/public/monocle-client-angular.js

```diff
@@ -348,8 +436,199 @@
 
 	'use strict';
 
+	exports.decode = exports.parse = __webpack_require__(5);
+	exports.encode = exports.stringify = __webpack_require__(6);
+
+
+/***/ },
+/* 5 */
+/***/ function(module, exports) {
```

Comment:
I know this is copied over... in the rest of the code the variables are expressive. EX: return instead of ret, but this section of the code the variables are cut which makes it harder to read.

Should we keep the same convention everywherE?